docs/scripts/translate-qa.py

```
@@ -0,0 +1,424 @@
+    parser = argparse.ArgumentParser(description="QA score Wails v3 translations")
+    parser.add_argument("--locale", default="all")
+    parser.add_argument("--json", action="store_true", help="Output full JSON")
+    parser.add_argument("--ai-verify", action="store_true",
+                        help="Use z.ai LLM to verify translation accuracy (requires ZAI_API_KEY env var)")
+    parser.add_argument("--ai-model", default=ZAI_DEFAULT_MODEL,
+                        help=f"z.ai model to use for verification (default: {ZAI_DEFAULT_MODEL})")
+    args = parser.parse_args()
+
+    if args.ai_verify and not ZAI_API_KEY:
+        print("⚠ --ai-verify requires ZAI_API_KEY environment variable to be set.", flush=True)
+        print(f"  Set it with: export ZAI_API_KEY=<your-key>", flush=True)
```

Ruff (0.15.12)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)